docs/writing-tests/python-handlers/index.md

```diff
@@ -6,10 +6,25 @@ requests made to the corresponding URL. This is hooked up to a route like
 treated as a handler file (note that this makes it easy to write unsafe
 handlers, particularly when running the server in a web-exposed setting).
 
-The Python files must define a single function `main` with the signature::
+The Python files must define a function named `main` with the signature:
```

Comment:
Can we directly reference the `request` and `response` types now #17418 has landed, rather than just having the TOC below?

Reply:
Excellent idea! Rebased and added a fixup commit. The prior version of this branch is available here:

https://github.com/web-platform-tests/wpt/compare/master...bocoup:docs-python-scripting-orig